Update repositories.json

kjecobra opened the pull request.

tr4wzified commented:
Can you use a  direct link to an image hosted on GitHub? imgur won't work. Additionally, please only use the tags defined here: https://github.com/wabbajack-tools/mod-lists/blob/master/allowed_tags.json

kjecobra commented:
I changed the image.. not sure what was meant by the tags. I removed the
tags I entered, and used tags in the list provided..    Do I need to update
anything on the repository side.

And not sure about the SFW NeverNude ..  the list provides NEVER NUDE, just
not build in bodyslide from the start.

On Sat, May 17, 2025 at 3:54 AM trawzified ***@***.***> wrote:

> *tr4wzified* left a comment (wabbajack-tools/mod-lists#3703)
> <https://github.com/wabbajack-tools/mod-lists/pull/3703#issuecomment-2888201219>
>
> Can you use a direct link to an image hosted on GitHub? imgur won't work.
> Additionally, please only use the tags defined here:
> https://github.com/wabbajack-tools/mod-lists/blob/master/allowed_tags.json
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wabbajack-tools/mod-lists/pull/3703#issuecomment-2888201219>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM2LN5E2AVOJCJWF5N6AIJL263TK3AVCNFSM6AAAAAB5JWMELSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQOBYGIYDCMRRHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
